src/rupali/assignment26/MaxDigit.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Assignment - 26 : 19th April'2021
+
+Program 1: Write a method which will return a string containing maximum digits.
+Input : {"P1u2r2va", "1K2omal", "9S4hivan4i4", "1K2r3ati", "4M4a5d5h1v1i93"}
+Output : 4M4a5d5h1v1i93
+ */
+package rupali.assignment26;
+
+public class MaxDigit {
+	String stringwitnmaxdigit(String[] input) {
+		int i=0,j,count=0,maxcount=0;
+		String ch1=null;
+		for(i=0;i<input.length;i++) {
+				for(j=0;j<input[i].length();j++) {
```

Comment:
count should be init with 0 for new word right, your logic break for below input, online 16 you should write count = 0

this will fail for String [] arr={"P1u2r2va", "1K2omal", "4M4a5d5h1v1i93234","9S4hivan4i4", "1K2r3ati", "4M4a5d5h1v1i93"};